### PR TITLE
examples/kubernetes: restart kube-dns in the background

### PR DESCRIPTION
**Pull request by aanm**
Signed-off-by: André Martins <andre@cilium.io>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/4777)

### CODE REVIEW

**Comment by aanm** [comment]
test-me-please

File: examples/kubernetes-ingress/scripts/06-install-kubedns.sh [hunk 1]
```diff
@@ -29,21 +29,14 @@ sed -e "s/127.0.0.1:/${dns_probes_ips[0]}:/" \
...
-
-kubectl create -f "${controller_file}" || true
+nohup kubectl delete -f "${controller_file}" 2>/dev/null || true && \
+kubectl delete -f "${svc_file}" 2>/dev/null || true && \
+kubectl delete -f "${sa_file}" 2>/dev/null || true && \
+kubectl delete -f "${cm_file}" 2>/dev/null || true && \
+kubectl create -f "${cm_file}" || true && \
+kubectl create -f "${sa_file}" || true && \
+kubectl create -f "${svc_file}" || true && \
+kubectl create -f "${controller_file}" || true &
```
Comment (ianvernon) [inline examples/kubernetes-ingress/scripts/06-install-kubedns.sh:39]: Is the extra ampersand at the end of this line supposed to be there?

Comment (raybejjani) [reply examples/kubernetes-ingress/scripts/06-install-kubedns.sh:39]: I think that's the backgrounding bit.

@aanm does this work correctly, though? Does the single `&` background all the delete/create calls? (i.e. does it need `()` around everything?)

Comment (ianvernon) [reply examples/kubernetes-ingress/scripts/06-install-kubedns.sh:39]: Ah, got it...

**Review by ianvernon — APPROVED** [review]

File: examples/kubernetes-ingress/scripts/06-install-kubedns.sh (hunk 1 above)
Comment (aanm) [reply examples/kubernetes-ingress/scripts/06-install-kubedns.sh:39]: @raybejjani I don't think it needs, I try it out by doing `nohup sleep 5s && echo foo &`

**Pull request merged**